docs/content/dataset_standards.md

```diff
@@ -177,10 +177,233 @@ The stepping data inside the episode group is divided into some required `datase
+                        <ul>
+                            <li class="dataset-white">_index_0</li>
+                            <li class="folder-white">_index_1_
+                            <ul>
+                            <li class="dataset-white"> _index_0 </li>
+                            <li class="dataset-white"> _index_1_ </li>
+                            </ul>
+                            </li>
+                        </ul>
+                    </li>
+                    <li class="dataset-white">terminations</li>
+                    <li class="dataset-white">truncations</li>
+                    <li class="dataset-white">rewards</li>
+                    <li class="folder-white">infos
+                    <ul class="white">
+                        <li class="dataset-white">infos_datasets</li>
+                        <li class="folder-white">infos_subgroup
+                        <ul class="white">
+                            <li class="dataset-white">more_datasets</li>
+                        </ul>
+                        </li>
+                    </ul>
+                    </li>
+                    <li class="folder-white">additional_groups
+                        <ul class="white">
+                            <li class="dataset-white">additional_datasets</li>
+                        </ul>
+                    </li>
+                </ul>
+            </li>
+            <li class="folder-white-closed">episode_1</li>
+            <li class="folder-white-closed">episode_2</li>
+            <ul class="white"><br></ul>
+            <li class="folder-white-closed">episode_id</li>
+        </ul>
+    </li>
+</ul>
+</div>
+
+Note how the `Tuple` space elements are assigned corresponding keys of the format `f"_index_{i}"` were `i` is their index in the `Tuple` space.
+
+
+The required `datasets` found in the episode groups correspond to the data involved in every Gymnasium step call `obs, rew, terminated, truncated, info = env.step(action)`: `observations`, `actions`, `rewards`, `terminations`, and `truncations`. These datasets are `np.ndarray` or nested groups of `np.ndarray` and other groups, depending on the observation and action spaces, and the space of all datasets under each required top-level episode key is equal to:
+
+- `actions`: `shape=(number_of_steps, action_space_component_shape)`. If the action or observation space is `Dict` or a `Tuple`, then the corresponding entry will be a group instead of a dataset. Within this group, there will be nested groups and datasets, as specified by the action and observation spaces. `Dict` and `Tuple` spaces are represented as groups, and `Box` and `Discrete` spaces are represented as datasets. All datasets at any level under the top-level key `actions` will have the same `number_of_steps`, but will vary in `action_space_component_shape` on for each particular action space component. For example, a `Dict` space may contain two `Box` spaces with different shapes.
```

Comment:
May be too late to change but number of steps is a long variable name, num_steps seems much not concise and easy for users to remember

Reply:
renamed